src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--length-28">
+      <div
+        class="calendar__item calendar__item--start-day-fri"
+      ></div>
```

Comment:
![image](https://github.com/mate-academy/layout_calendar/assets/55200530/af4b279e-612e-4917-81b3-104bd53aedd9)

not on item